Source/SwiftLintFramework/Rules/Idiomatic/VoidFunctionInTernaryConditionRule.swift

```diff
@@ -34,6 +34,40 @@ public struct VoidFunctionInTernaryConditionRule: ConfigurationProviderRule, Aut
             array.map { elem in
                 elem.isEmpty() ? .emptyValue() : .number(elem)
             }
+            """),
+            Example("""
+            func compute(data: [Int]) -> Int {
+                data.isEmpty ? 0 : expensiveComputation(data)
+            }
+            """),
+            Example("""
+            func get() -> Int { 1 }
+            func get(from elements: [Int]) -> Int { elements[0] }
+            func f(elements: [Int]) -> Int {
+                elements.isEmpty ? get() : get(from: elements)
+            }
```

Comment:
Looks like the case tested in this example is already covered by the one above. Do you still want to keep it?